test/test_ws.py

```diff
@@ -0,0 +1,127 @@
+import time
+import json
+import base64
+import pytest
+
+from common import running_on_ci
+message = []
+
+def test_ws_connection(socketio):
+    print('my sid is', socketio.sid)
+    assert socketio.sid is not None
+
+def test_list(socketio):
+    global message
+    socketio.on('message', message_handler)
+    socketio.emit('command', 'list')
+    time.sleep(.2)
+    print (message)
+    assert any("list" in i for i in message)
+    assert any("Ports" in i for i in message)
+    assert any("Network" in i for i in message)
+
+# NOTE run the following tests on linux with a board connected to the PC and with this sketch on it: https://gist.github.com/Protoneer/96db95bfb87c3befe46e
```

Comment:
In this case where the sketch is trivial (meaning there is no benefit to maintaining it as a separate project), I would suggest adding it to this repository, either as a file or even just a comment here. That will avoid any potential for annoyance in the future if the 3rd party Gist happens to be deleted.